packages/utils/typescript/lib/generators/common/models/attributes.js

```diff
@@ -114,21 +114,21 @@ const getAttributeModifiers = (attribute) => {
       throw new Error('typeof min/max values mismatch');
     }
 
-    const typeofMinMax = (max && typeofMax) ?? (min && typeofMin);
     let typeKeyword;
 
-    // Determines type keyword (string/number) based on min/max options, throws error for invalid types
-    switch (typeofMinMax) {
-      case 'string':
-        typeKeyword = ts.SyntaxKind.StringKeyword;
-        break;
-      case 'number':
-        typeKeyword = ts.SyntaxKind.NumberKeyword;
-        break;
-      default:
-        throw new Error(
-          `Invalid data type for min/max options. Must be string or number, but found ${typeofMinMax}`
-        );
+    // use 'string'
+    if (typeofMin === 'string' || typeofMax === 'string') {
+      typeKeyword = ts.SyntaxKind.StringKeyword;
+    }
+    // use 'number'
+    else if (typeofMin === 'number' || typeofMax === 'number') {
```

Comment:
```suggestion
    if (typeofMin === 'string') {
      typeKeyword = ts.SyntaxKind.StringKeyword;
    }
    // use 'number'
    else if (typeofMin === 'number') {
```
Might be clearer like this, I originally asked "what if they're different" because I didn't notice the hidden lines above checking for that case.

Reply:
Except if you want to have 4 ifs here? :thinking: